ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -23,6 +23,8 @@
 from ocs_ci.utility.aws import update_config_from_s3
 from ocs_ci.utility.utils import load_auth_config
 
+# from ocs_ci.ocs.cluster import CephCluster
```

Comment:
if this is not needed, pleas delete the lines

Reply:
Done